index.html

```diff
@@ -759,6 +760,14 @@
                 </ol>
               </li>
             </ol>
+            <div class="note">
+              To avoid leaking information that could fingerprint the user, the
+              user agent SHOULD assign the <var>callbackId</var> based on the
+              identity of the <a>media element</a> alone, and not the identity
+              of the browser profile or any <a>remote playback device</a>.  For
+              example, the user agent could base the <var>callbackId</var> on
+              the timestamp the <a>media element</a> was created.
```

Comment:
Fair point. It might be more likely having one callback per many elements compared to many callbacks for a single element. The spec doesn't guarantee any cross-element uniqueness of the ids at the moment though.

@jernoble @mounirlamouri WDYT?